src/stylesheets/_fonts.scss

```diff
@@ -39,3 +39,59 @@
     url("~fonts/montserrat-v15-latin-500italic.ttf") format("truetype"),
     url("~fonts/montserrat-v15-latin-500italic.svg#Montserrat") format("svg"); /* Legacy iOS */
 }
+
+/* source-code-pro-regular - latin */
+@font-face {
+  font-family: "Source Code Pro";
+  font-style: normal;
+  font-weight: 400;
+  src: url("../fonts/source-code-pro-v13-latin-regular.eot"); /* IE9 Compat Modes */
+  src: local(""),
+    url("../fonts/source-code-pro-v13-latin-regular.eot?#iefix") format("embedded-opentype"), /* IE6-IE8 */
+    url("../fonts/source-code-pro-v13-latin-regular.woff2") format("woff2"), /* Super Modern Browsers */
+    url("../fonts/source-code-pro-v13-latin-regular.woff") format("woff"), /* Modern Browsers */
+    url("../fonts/source-code-pro-v13-latin-regular.ttf") format("truetype"), /* Safari, Android, iOS */
+    url("../fonts/source-code-pro-v13-latin-regular.svg#SourceCodePro") format("svg"); /* Legacy iOS */
```

Comment:
All these imports can be done from `~fonts/...`. The power and magic of having our root at `src`!